doc/source/user/building.rst

```diff
@@ -274,16 +290,6 @@ variables. The default value is ``openblas64_,openblas_ilp64``.
    The symbol suffix avoids the symbol name clashes between 32-bit and
    64-bit BLAS/LAPACK libraries.
 
-
-Supplying additional compiler flags
------------------------------------
-
-Additional compiler flags can be supplied by setting the ``OPT``,
-``FOPT`` (for Fortran), and ``CC`` environment variables.
-When providing options that should improve the performance of the code
-ensure that you also set ``-DNDEBUG`` so that debugging code is not
-executed.
```

Comment:
Perhaps worth mentioning the environment variables and linking to the meson FAQ? https://mesonbuild.com/howtox.html#set-extra-compiler-and-linker-flags-from-the-outside-when-eg-building-distro-packages